vmdb/app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -162,6 +162,20 @@ def save_containers_inventory(container_group, hashes, target = nil)
     store_ids_for_new_records(container_group.containers, hashes, :ems_ref)
   end
 
+  def save_container_node_conditions_inventory(container_node, hashes, target = nil)
+    return if hashes.nil?
+
+    container_node.container_node_conditions(true)
+    deletes = if target.kind_of?(ExtManagementSystem)
+                container_node.container_node_conditions.dup
+              else
+                []
+              end
+
+    save_inventory_multi(:container_node_conditions, container_node, hashes, deletes, [:name])
+    store_ids_for_new_records(container_node.container_node_conditions, hashes, :name)
```

Comment:
@Fryguy these are both already scoped per parent (`container_node`) right? So `:name` should be enough.